web/viewer.css

```diff
@@ -582,6 +583,11 @@ body {
   background-color: var(--toolbar-bg-color);
 }
 
+#editorStampParamsToolbar {
+  inset-inline-end: 96px;
+  background-color: var(--toolbar-bg-color);
+}
+
```

Comment:
These rules looks unused, can't we wait with adding this until it's actually needed?